src/ripple/app/tests/Offer.test.cpp

```diff
@@ -41,6 +42,15 @@ class Offer_test : public beast::unit_test::suite
         return env.current()->info().parentCloseTime.time_since_epoch().count();
     }
 
+    static auto
+    xrpMinusFee (jtx::Env const& env, std::int64_t xrpAmount)
+        -> jtx::PrettyAmount
+    {
+        using namespace jtx;
+        auto feeDrops = env.current ()->fees ().base;
+        return drops (dropsPerXRP<std::int64_t>::value * xrpAmount - feeDrops);
```

Comment:
We've started proliferating types again. We have `XRPAmount` and `XRP_t` and `drops` and `STAmount` still and ugh... we need to consolidate.